test/mandelbrot.cxx

```diff
@@ -78,6 +78,12 @@ void Drawing_Window::update_label() {
 }
 
 void Drawing_Area::draw() {
+  if (!dx) {
+    dx = Fl::box_dx(box());
+    dy = Fl::box_dy(box());
+    W -= 2*dx;
+    H -= 2*dy;
+  }
```

Comment:
Is there a specific reason why you use `2*dx` and `2*dy` rather than `Fl::box_dw(box())` and `Fl::box_dh(box())`, respectively? Even if this would in almost all cases be the same value, it would IMHO be more consistent to use the "official" box values.

This could be used later as well (see note below).